### ISSUE TITLE
Autograd and backpropagation

### ISSUE THREAD

**Issue by user-aqm** [wavefrontshaping/complexPyTorch#35]
hi
I am a beginner to complex valued networks, My program uses real-valued loss functions and the network does not converge and I have a question: What kind of loss function should guide the network for complex valued networks? Is real or complex loss more appropriate? This must have something to do with how torch backspreads now. My understanding in this area is insufficient, although torch gives the specific calculation procedure.

![autograd](https://github.com/wavefrontshaping/complexPyTorch/assets/59562538/182ea079-cc51-4958-91ad-bde184e40e86)

I am looking forward to your reply. Thank you very much.
